dev/com.ibm.ws.jaxrs.2.1_fat_extended/publish/servers/jaxrs21.fat.jsonbContextResolver/bootstrap.properties

```diff
@@ -0,0 +1,8 @@
+com.ibm.ws.logging.trace.specification=*=info:\
+com.ibm.ws.jaxrs20.*=all:\
```

Comment:
is this extra trace intentional?

Reply:
We could probably remove it.  I copied it from the other JSON-B test server.